### PR TITLE
Issue #256: SuppressionPatchXpathFilter: HideUtilityClassConstructor's context strategy

### PR DESCRIPTION
**Pull request by HuGanghui**
Issue #256: SuppressionPatchXpathFilter: HideUtilityClassConstructor's context strategy

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/HideUtilityClassConstructor/newline/defaultContextConfig.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+        "https://checkstyle.org/dtds/configuration_1_3.dtd">
+<module name="Checker">
+    <module name="TreeWalker">
+        <module name="HideUtilityClassConstructor"/>
+
+        <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionPatchXpathFilter">
+            <property name="file" value="${tp}/defaultContext.patch" />
+            <property name="strategy" value="newline" />
+            <property name="supportContextStrategyChecks" value="HideUtilityClassConstructorCheck" />
```
Comment (rdiachenko) [inline src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/HideUtilityClassConstructor/newline/defaultContextConfig.xml:12]: It is not a context strategy.

File: src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/HideUtilityClassConstructor/patchedline/defaultContextConfig.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+        "https://checkstyle.org/dtds/configuration_1_3.dtd">
+<module name="Checker">
+    <module name="TreeWalker">
+        <module name="HideUtilityClassConstructor"/>
+
+        <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionPatchXpathFilter">
+            <property name="file" value="${tp}/defaultContext.patch" />
+            <property name="strategy" value="patchedline" />
+            <property name="supportContextStrategyChecks" value="HideUtilityClassConstructorCheck" />
```
Comment (rdiachenko) [inline src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/HideUtilityClassConstructor/patchedline/defaultContextConfig.xml:12]: Same as above.

File: src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/HideUtilityClassConstructor/newline/defaultContextConfig.xml (hunk 1 above)
Comment (HuGanghui) [reply src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/HideUtilityClassConstructor/newline/defaultContextConfig.xml]: done

File: src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/HideUtilityClassConstructor/patchedline/defaultContextConfig.xml (hunk 2 above)
Comment (HuGanghui) [reply src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/HideUtilityClassConstructor/patchedline/defaultContextConfig.xml]: done

Comment (HuGanghui) [reply src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/HideUtilityClassConstructor/patchedline/defaultContextConfig.xml]: done

**Pull request merged**